exchange/exchange.go

```diff
@@ -57,6 +59,16 @@ func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *con
 }
 
 func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidRequest, usersyncs IdFetcher) (*openrtb.BidResponse, error) {
+	// Snapshot of resolved bid request for debug if test request
+	var resolvedRequest openrtb.RawJSON
```

Comment:
This could just be:
```go
if resolvedRequest, err := json.Marshal(bidRequest); err != nil {
    glog.Errorf("Error marshalling bid request for debug: %v", err)
}
```
but I wanted to only spend the time doing this in test requests.

Reply:
I believe we can use `json.RawMessage` in place of `openrtb.RawJSON`. I have also heard that the openrtb library is planning on switching to this to have a more consistent handling of raw JSON blobs.

Reply:
Sounds good. They both are just aliases for `byte[]` I believe but better to use the one in the go stdlib.